bats/tests/extensions/allow-list.bats

```diff
@@ -0,0 +1,107 @@
+load '../helpers/load'
+
+setup() {
+    TESTDATA_DIR="${PATH_BATS_ROOT}/tests/extensions/testdata/"
+
+    if using_windows_exe; then
+        TESTDATA_DIR_CLI="$(wslpath -m "${TESTDATA_DIR}")"
+    else
+        TESTDATA_DIR_CLI="${TESTDATA_DIR}"
+    fi
+
+    if using_containerd; then
+        namespace_arg=('--namespace=rancher-desktop-extensions')
+    else
+        namespace_arg=()
+    fi
+}
+
+write_allow_list() { # list
```

Comment:
Maybe a test that passes more than one arg to this function, because currently it only handles 0 or 1 values in the list?

Reply:
It takes a single argument that is the JSON encoded list.

Reply:
I realized that, but none of the args contain a comma

Reply:
Ah, I thought I had one. Will fix.